packages/utils/telemetry-utils/src/index.ts

```diff
@@ -56,6 +56,7 @@ export {
 	IPerformanceEventMarkers,
 	ITelemetryLoggerPropertyBag,
 	ITelemetryLoggerPropertyBags,
+	MultiSinkLoggerProperties,
```

Comment:
Is there a reason this is being exported now? Just curious since it was the only one.

Reply:
I added this interface to encapsulate documentation for the props passed to `createMultiSinkLogger`. I left a brief explanation in the PR description.